taskfile/ast/graph.go

```diff
@@ -116,14 +116,14 @@ func (tfg *TaskfileGraph) Merge() (*Taskfile, error) {
 		return nil, err
 	}
 
-	_ = rootVertex.Taskfile.Tasks.Range(func(name string, task *Task) error {
+	// TODO: I don't think this is necessary anymore
```

Comment:
Do you want to resolve / check this TODO in another PR ?

Reply:
This was a note to myself and I just forgot to come back to it. `Task.Task` is populated when the `Tasks` parent type is decoded so this is no longer needed. I've rebased the PR and removed this section.